detections/application/splunk_risky_command_abuse_ml_detection.yml

```diff
@@ -0,0 +1,67 @@
+name: Splunk Risky Command Abuse ML Detection
+id: 19d0146c-2eae-4e53-8d39-1198a78fa9ca
+version: 1
+date: '2022-05-27'
+author: Abhinav Mishra, Kumar Sharad and Xiao Lin, Splunk
+type: Anomaly
+datamodel:
+- Splunk_Audit
+description: 'This detection utilize machine learning model named "risky_command_abuse" trained from "Splunk 
+  Risky Command Abuse ML Model Baseline". It should be scheduled to run every hour 
+  to detect whether a user has run searches containing risky keywords with abnormal long running time in the 
+  past two hours, comparing with his/her past seven days history. '
+search: '| tstats values(Search_Activity.user) AS user, sum(Search_Activity.total_run_time) AS run_time,
+  values(Search_Activity.search) as searches, count 
+  FROM datamodel=Splunk_Audit.Search_Activity WHERE (Search_Activity.user!="") 
+  AND (Search_Activity.total_run_time>1) AND (earliest=-2h@h latest=now) 
+  AND (Search_Activity.search IN ("*| runshellscript *", "*| collect *","*| delete *", "*| fit *", "*| outputcsv *", 
+  "*| outputlookup *", "*| run *", "*| script *", "*| sendalert *", "*| sendemail *", "*| tscollect *")) 
+  AND (Search_Activity.search_type=adhoc) AND (Search_Activity.user!=splunk-system-user) 
+  BY _time, Search_Activity.user span=1h 
+  | apply risky_command_abuse
+  | fields _time, user, searches, run_time, IsOutlier(run_time)
+  | rename IsOutlier(run_time) as isOutlier, _time as timestamp
+  | where isOutlier>0.5 | `splunk_risky_command_abuse_ml_detection_filter`'
```

Comment:
I would explain in the description why "isOutlier>0.5" the threshold set by default.

Reply:
added